### DISCUSSION TITLE
how to create a discussion as the organisation itself?

### DISCUSSION THREAD

**Discussion by hp8wvvvgnj6asjm7** [community/community#63328] [Discussions]
### Select Topic Area

Question

### Body

Hello,

I created an organisation from my private profile, but then found out later that it is possible to turn your private profile into an org.

1. In my discussion page on the organsiation repository,  I can only post from my private profile. 
2. I wonder if there is a difference between an organisation created in a profile, or a profile turned into an org.
3. if turned into an org, which profile would you post discussions as then?

Assuming that when turning your private profile into an org, the private profile would disappear, which then should be able to post as the org.

**Comment by Jaycon80** [comment]
[](url)https://github.com/orgs/community/discussions/63328
